backend/pluto/pluto/_internal/dash/utils.py

```diff
@@ -49,7 +49,7 @@ def total_amount_in_months(
                 obj.exp_date if isinstance(obj, Expense) else obj.inc_date
             )
             if isinstance(obj_date, str):
-                obj_date = datetime.strptime(obj_date, "%d/%m/%Y")
+                obj_date = datetime.strptime(obj_date, "%Y-%m-%d")
```

Comment:
O objetivo de ser `%d/%m/%Y` era para facilitar o entendimento do usuário do Brasil  em ver as datas no **gráfico**. Se colocar assim, acho que as datas no gráfico ficarão ruins. Acho melhor manter o original.

Reply:
Please see my other related comment. The BR date format is set manually in the dash utils stuff.